src/Symfony/Component/Form/Tests/FormErrorSerializableTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Symfony\Component\Form\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+use Symfony\Component\Form\FormBuilder;
+use Symfony\Component\Form\FormError;
+use Symfony\Component\Validator\ConstraintViolation;
+
+class FormErrorSerializableTest extends TestCase
+{
+    public function testSerializeFormError()
+    {
+        if (!class_exists(ConstraintViolation::class)) {
+            $this->markTestSkipped('Validator component required.');
+        }
+        $formBuilder = new FormBuilder(
+            'form',
+            null,
+            new EventDispatcher(),
+            $this->getMockBuilder('Symfony\Component\Form\FormFactoryInterface')->getMock(),
+            array()
+        );
+
+        $form = $formBuilder->getForm();
+        $simpleXMLElement = new \SimpleXMLElement('<foo></foo>');
+        $cause = new ConstraintViolation('Error 1!', null, array(), $simpleXMLElement, '', null, null, '');
+        $form->addError(new FormError('Error 1!', null, array(), null, $cause));
+        $formErrors = $form->getErrors();
+        $specificFormError = $formErrors->current();
+        dump($cause->serialize());
```

Comment:
?

Reply:
Sure, I can use the FormError directly without instantiate a form object.

line 32: try to call the serialize function as on Travis ( see below ) for some reason the ConstraintViolation class has been overrided.

https://travis-ci.org/symfony/symfony/jobs/313999394